master/reference/felix/configuration.md

```diff
@@ -30,6 +30,8 @@ The full list of parameters which can be set is as follows.
 | StartupCleanupDelay                     | FELIX_STARTUPCLEANUPDELAY               | 30                                   | Delay, in seconds, before felix does its start-of-day cleanup to remove orphaned iptables chains and ipsets.  Before the first cleanup, felix operates in "graceful restart" mode,  during which it preserves any pre-existing chains and ipsets. In a large deployment you may want to increase this value to give felix more time to load the initial snapshot from etcd before cleaning up.  |
 | PrometheusMetricsEnabled                | FELIX_PROMETHEUSMETRICSENABLED          | "false"                              | Set to "true" to enable the experimental Prometheus metrics server in Felix.  |
 | PrometheusMetricsPort                   | FELIX_PROMETHEUSMETRICSPORT             | 9091                                 | Experimental: TCP port that the Prometheus metrics server should bind to.  |
+| PrometheusGoMetricsEnabled              | FELIX_PROMETHEUSGOMETRICSENABLED        | "true"                               | Set to "false" to prevent the experimental Prometheus Go(Lang) specific metrics gathering in Felix.  |
```

Comment:
How about this sort of phrasing: `Set to "false" to disable Go runtime metrics collection, which the Prometheus client does by default.  This reduces the number of metrics reported, reducing Prometheus load.`